src/sql/workbench/api/node/extHostAccountManagement.ts

```diff
@@ -64,6 +64,33 @@ export class ExtHostAccountManagement extends ExtHostAccountManagementShape {
 		this._proxy.$accountUpdated(updatedAccount);
 	}
 
+	public $getAllAccounts(): Thenable<sqlops.AccountWithProviderHandle[]> {
```

Comment:
I'd recommend adding a test for this method in `src/sqltest/workbench/api/extHostAccountManagement.test.ts`